cmd/lakefs/cmd/migrate.go

```diff
@@ -55,46 +53,17 @@ var upCmd = &cobra.Command{
 	Short: "Apply all up migrations",
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg := loadConfig()
-		err := db.MigrateUp(cfg.GetDatabaseParams())
+		err := db.MigrateUp(cfg.GetDatabaseParams(), cfg)
 		if err != nil {
 			fmt.Printf("Failed to setup DB: %s\n", err)
 			os.Exit(1)
 		}
 	},
 }
 
-var gotoCmd = &cobra.Command{
```

Comment:
I think that as long as user didn't perform the migrate, we should be able to use this functionality. Maybe this should be removed post KV version